package/CHANGELOG

```diff
@@ -111,6 +111,7 @@ Enhancements
   * Improve the distance search in water bridge analysis with capped_distance (PR #2480)
   * Added weights_groupselections option in RMSD for mapping custom weights 
     to groupselections (PR #2610, Issue #2429)
+  * Added user warning for CRYST1 cryo-em structures (PR #2645, Issue #2599)
```

Comment:
Two things here:
1) Since this does more than just add a warning I'd be more explicit here, something along the lines of "Checks for cryo-em 1 A^3 default CRYST1 record, disabling setting of box dimensions if found (Issue #2599)".
2) This probably should go under "fixes". Also as per the CHANGELOG rules, entries should be added "newest first", i.e. at the top of the list.